### PR TITLE
tweak enterprise migration, improve backup docs

### PR DESCRIPTION
**Pull request by beckettsean**
@rkuchan @jackzampolin

### CODE REVIEW

**Comment by beckettsean** [comment]
Closes https://github.com/influxdata/docs.influxdata.com/issues/681, too, @kostasb

**Comment by kostasb** [comment]
Looks great.

**Comment by jackzampolin** [comment]
:+1:

File: content/influxdb/v1.0/concepts/glossary.md [hunk 1]
```diff
@@ -92,7 +92,8 @@ Measurements are strings.
 Related entries: [field](/influxdb/v1.0/concepts/glossary/#field), [series](/influxdb/v1.0/concepts/glossary/#series)
 
 ## metastore
-Contains internal information about the status of the system. That includes user information, database and shard metadata, and which retention policies are enabled.
+Contains internal information about the status of the system. 
+The metastore containts the user information, databases, retention policies, shard metadata, continuous queries, and subscriptions.
```
Comment (rkuchan) [inline content/influxdb/v1.0/concepts/glossary.md]: containts -> contains

**Comment by rkuchan** [comment]
One small fix -otherwise 👍

**Comment by beckettsean** [comment]
fixes #681

**Pull request merged**